docs/Overview/SupportedDataSources.md

```diff
@@ -5,61 +5,77 @@ description: >
-| refs                  |                | default      |                |         |         |         |         |         |
-| refs_commits_diffs    |                |              |                |         |         |         | default |         |
-| refs_issues_diffs     |                |              |                |         |         |         | default |         |
-| ref_pr_cherry_picks   |                |              |                |         |         |         | default |         |
-| repos                 |                |              | default        | default |         |         |         |         |
-| repo_commits          |                | default      | default        |         |         |         |         |         |
-| board_repos           |                |              |                |         |         |         |         |         |
-| issue_commits         |                |              |                |         |         |         |         |         |
-| issue_repo_commits    |                |              |                |         |         |         |         |         |
-| pull_request_issues   |                |              |                |         |         |         |         |         |
-| refs_issues_diffs     |                |              |                |         |         |         |         |         |
-| boards                |                |              | default        |         |         | default |         | default |
-| board_issues          |                |              | default        |         |         | default |         | default |
-| issue_changelogs      |                |              |                |         |         | default |         | default |
-| issues                |                |              | default        |         |         | default |         | default |
-| issue_comments        |                |              |                |         |         | default |         | default |
-| issue_labels          |                |              | default        |         |         |         |         |         |
-| sprints               |                |              |                |         |         | default |         | default |
-| issue_worklogs        |                |              |                |         |         | default |         | default |
-| users o               |                |              | default        |         |         | default |         | default |
-| builds                |                |              |                |         | default |         |         |         |
-| jobs                  |                |              |                |         | default |         |         |         |
-
-
 
+This table shows the entities collected by each plugin. Domain layer entities in this table are consistent with the entities [here](/DataModels/DevLakeDomainLayerSchema.md).
+✅ : default collected.
```

Comment:
collect by default
collect not by default. Can you explain how to collect these entities?